sdk/python/packages/flet/src/flet/controls/core/list_view.py

```diff
@@ -54,6 +54,9 @@ class ListView(LayoutControl, ScrollableControl, AdaptiveControl):
     """
     A fixed height or width (when [`horizontal`][(c).] is `True`)
     of an item to optimize rendering.
+
+    Note: This property has effect only when [`build_controls_on_demand`][(c).]
+        is `True` or [`spacing`][(c).] is `0`.
```

Comment:
<img width="818" height="324" alt="image" src="https://github.com/user-attachments/assets/2893b6ce-82c9-4d80-be28-9f559f9de6a4" />

"Note:" must be alone one one line.
```
Note:
    text here...
```